test/testdata/rbi/array.rb

```diff
@@ -35,3 +35,9 @@
 # errors
 
 T::Array[Float].new.sum.nan? # error: Method `nan?` does not exist on `Integer` component of `T.any(Integer, Float)`
+
+# Sample
+T.assert_type!([1, 2].sample, Integer)
+T.assert_type!([1, 2].sample, random: Random.new, Integer)
```

Comment:
Did you mean this to be like so?
```suggestion
T.assert_type!([1, 2].sample(random: Random.new), Integer)
```